### DISCUSSION TITLE
raspberry pi relay board

### DISCUSSION THREAD

**Discussion by cinaeds** [duggabe/gr-control#6] [Q&A] [answered]
Hi Barry - any recommendations for a relay board for the raspberry pi? I have a raspberry pi4b but I don't know anything about the hardware hats. 

-- Cinaed

**Comment by duggabe** [comment, marked as answer]
The https://thepihut.com/products/raspberry-pi-relay-board is the one I am using. Note that I added jumpers to use different pins, rather than the ones on the board. See the notes in https://github.com/duggabe/gr-control/blob/main/relay_sequencer.py
